earth_enterprise/rpms/opengee-server/post-install.sh

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 set +x
+umask 002
```

Comment:
I am assuming this is the only rpm script that currently requires this added, and I see both install_xx scripts have it now too.

Reply:
Correct - the fusion post_install.sh does not create any files.  While it does sometimes create /gevol, I don't believe this is much of an issue as there's a lot of user administration around /gevol management that will probably require documentation and/or modification to other scripts (i.e., geconfigureassetroot) if un-sane umask causes a problem there.